modules/sql-engine/src/test/resources/mapping/table_identity.test

```diff
@@ -4,29 +4,30 @@ SELECT * FROM nt1_n1, nt2_n2
 Fragment#0 root
   executionNodes: [N0]
   remoteFragments: [1, 2]
-  exchangeSourceNodes: {1=[N1], 2=[N2]}
+  exchangeSourceNodes: {1=[N2], 2=[N1]}
   tree:
-    NestedLoopJoin
-      Receiver(sourceFragment=1, exchange=1, distribution=single)
-      Receiver(sourceFragment=2, exchange=2, distribution=single)
+    Project
```

Comment:
This project is a result of changing tables join order.
Seems, it can be pushed into join operator at execution phase.

Reply:
https://issues.apache.org/jira/browse/IGNITE-24196